MobileCenterCrashes/MobileCenterCrashes/Internals/Util/MSErrorLogFormatter.m

```diff
@@ -645,15 +645,16 @@ + (NSString *)anonymizedPathFromPath:(NSString *)path {
 
   if (([path length] > 0) && [path hasPrefix:@"/Users/"]) {
     NSError *error = nil;
-    NSRegularExpression *regex =
-        [NSRegularExpression regularExpressionWithPattern:@"(/Users/[^/]+/)" options:0 error:&error];
+    NSString *regexPattern = @"(/Users/[^/]+/)";
+    NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:regexPattern options:0 error:&error];
+    if (!regex) {
+      MSLogError([MSCrashes logTag], @"Couldn't create regular expression with pattern\"%@\": %@", regexPattern, error.localizedDescription);
+      return path;
```

Comment:
I think there could be a chance here to return a user identified path and we must really avoid that. Should probably return `anonymizedProcessPath` in this case, even if empty.